app/utils.py

```diff
@@ -768,30 +768,14 @@ def get_captcha_html(self, error_code=None, use_ssl=False):
 
         return captcha.get_display_html(
             public_key=config.get('captcha_public_key'),
```

Comment:
As discussed offline, rename public_key to site_key. Also change this code to use 'captcha_site_key' config.